src/api/entities/Asset/TransferRestrictions/ClaimCount.ts

```diff
@@ -36,6 +36,9 @@ export class ClaimCount extends TransferRestrictionBase<TransferRestrictionType.
   /**
    * Sets all Claim Count Transfer Restrictions on this Asset
    *
+   * @note this method set exempted Identities for restrictions as well. If an Identity is currently exempted from a Claim Count Transfer Restriction
+   * but not passed into this call then they will be removed
+   *
```

Comment:
```suggestion
   * @note this method sets exempted Identities for restrictions as well. If an Identity is currently exempted from a Claim Count Transfer Restriction
   * but not passed into this call, then it will be removed
   *
```

Reply:
There is another very similar comment on the percentage side that should also be addressed